react-app/src/types/LogType.tsx

```diff
@@ -8,9 +8,13 @@ export type Log = {
   summary: string;
   type: "MENTOR" | "TUTOR";
   student_id: string;
-  student_name: string;
+  id: string;
+  start_time: string;
+  end_time: string;
+  collapse: boolean;
```

Comment:
I don't think we should store collapse in the log object. Collapse is only used for the session log page to tell if the log should be collapsed or not, and it is changed by the user. When the user loads the session log page, collapsed should always be false and then they can change it if they choose, so I think it should be stored locally rather than in the database